doc/generate_cmake_rst.py

```diff
@@ -37,11 +37,12 @@
 import os
 import re
 import sys
+import json
 
 '''Simple superficial API doc generator for .cmake files'''
 
 
-def crawl_for_cmake(path):
+def crawl_for_cmake(path, excluded_files=[]):
```

Comment:
The default value should not be `[]` due to side effects in multiple invocations. Please use `None` instead.